### ISSUE TITLE
repeating the *clean_names()* step

### ISSUE THREAD

**Issue by josiecooke** [UM-R-for-EnvSci-Registered-Student/General-Discussion#2]
I was wondering if it is always required to use the *clean_names()* function when I import a .csv table that was coded and saved with data that had already been "cleaned"?

For example in Assignment 2, Exercise 1 we made a table of monthly max, min and mean temperatures (which had been "cleaned") and saved. However in Exercise 2 when I import that previously saved .csv file the data comes back using spaces, capitol letters, etc. and needs to be cleaned again.

@peperg I am not sure if this is an action that needs to be repeated each time, or if I may have incorrectly saved my .csv file. Thanks!

**Comment by peperg** [comment]
Hey Josie,

no, you shouldn't have to, if you have cleaned the names and then save that as a .csv, when you import it in a different script, later on, it should import with the good (clean names). I have cloned your repo and checked and it all looks good to me. The file loads on script 2 with the good, clean names. Perhaps the "original" version of the object was checked? Always make sure to re-start the R session in a new script as well, that is the kind of issue that could result from having "old" versions of objects living in the environment.

So, in summary. No, if you have saved an object (and even written it as a csv) after applying the clean_names() function, those changes should persist.

Pepe
